packages/valist-web/pages/[account]/[project]/launch.tsx

```diff
@@ -0,0 +1,55 @@
+import type { NextPage } from 'next';
+import { useContext, useEffect } from 'react';
+import { useNetwork } from 'wagmi';
+import { useRouter } from 'next/router';
+import useSWRImmutable from 'swr/immutable';
+import { useQuery } from '@apollo/client';
+import { Layout } from '@/components/Layout';
+import { ValistContext } from '@/components/ValistProvider';
+import query from '@/graphql/ProjectPage.graphql';
+import {
+  Loader,
+  Center,
+} from '@mantine/core';
+
+const ProjectPage: NextPage = () => {
+  const { chain } = useNetwork();
+
+  const router = useRouter();
+  const valist = useContext(ValistContext);
+
+  const accountName = `${router.query.account}`;
+  const accountId = valist.generateID(chain?.id ?? 0, accountName);
```

Comment:
```suggestion
  const accountId = valist.generateID(chain?.id ?? 137, accountName);
```

We'll need this to default to polygon :D